frontend/src/App.tsx

```diff
@@ -97,7 +103,7 @@ interface Completion {
   logprobs: string;
   notes: string;
 }
-type Rating = 'clear' | 'ok' | 'unclear' | 'wrong' | '' | 'demo' | 'acceptable' | 'no progress' | 'catastrophic';
+type RatingType = 'clear' | 'ok' | 'unclear' | 'wrong' | '' | 'demo' | 'acceptable' | 'no progress' | 'catastrophic';
```

Comment:
```suggestion
type RatingType = 'demo' | 'acceptable' | 'no progress' |'harmful'| 'catastrophic';
```
The first options were just while I was porting from the old rating system to the new